CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Unreleased
 
+# 19.1.0
+
+* Adds config option that allows apps to change the default behaviour of intercepting 401 requests, by including an initialiser: `GDS::SSO.config { |config| config.intercept_401_responses = false }`.
+* Added AuthorisedUserConstraint class so that consumer apps can easily add a permission based constraints.
+* Adds depracation warning for PermissionDeniedException to be replaced by PermissionDeniedError
+* Extracted User authorisation logic residing in ControllerMethods class into a separate class `AuthoriseUser` so it can be reused by both the ControllerMethods and the AuthorisedUserConstraint
```

Comment:
```suggestion
```

Since this is an implementation detail and not a class we expect consumers to use we don't need to put it in the changelog.